src/framegrab/grabber.py

```diff
@@ -666,9 +669,44 @@ def __init__(self, config: dict):
         thread.daemon = True
         thread.start()
 
+    def _substitute_rtsp_password(self, config: dict) -> dict:
+        """
+        Substitutes the password placeholder in the rtsp_url with the actual password
+        from an environment variable.
+        The URL format should take this format
+            Ex: rtsp://admin:{{RTSP_PASSWORD}}@10.0.0.0/cam/realmonitor?channel=1&subtype=0
```

Comment:
```suggestion
            Ex: rtsp://admin:{{MY_PASSWORD}}@10.0.0.0/cam/realmonitor?channel=1&subtype=0
```

Using `RTSP_PASSWORD` as the variable name can be confusing, because it seems like that is the exact variable name required.